src/Oracles/Setting.hs

```diff
@@ -130,7 +130,7 @@ settingList key = fmap words $ lookupValueOrError configFile $ case key of
     ConfCppArgs       stage -> "conf-cpp-args-"        ++ stageString stage
     ConfGccLinkerArgs stage -> "conf-gcc-linker-args-" ++ stageString stage
     ConfLdLinkerArgs  stage -> "conf-ld-linker-args-"  ++ stageString stage
-    HsCppArgs               -> "hs-cpp-args"
+    ConfHsCppArgs           -> "conf-hs-cpp-args"
```

Comment:
Why rename this flag? The configure script calls it `HaskellCPPArgs`. If you think the current naming is inconsistent, I suggest to fix the configure script first and then mirror the change in Hadrian.

Reply:
Reverting this in an upcoming commit.